### DISCUSSION TITLE
baseUrl change config error appears

### DISCUSSION THREAD

**Discussion by KInGATiLLa** [facebook/docusaurus#4073] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17881842/105143402-c36d1680-5b36-11eb-925f-23532ea17090.png)

Docusaurus version: 2.0.0-alpha.70

When I try to change baseUrl in config, then it doesn't correct render in production build serving from localhost.
How to fix it?

**Comment by slorber** [comment, marked as answer]
hi,

if you use the baseurl setting you need to ensure the proper file-system tree on your local computer.

If you use `baseUrl: '/subpath/'`, then move the `website/build folder in` `website/subpath` and serve the `website` folder.

I can't help you more without seeing your source code and a live version of your failing deployment.
